cloud-controller-manager/do/loadbalancers_test.go

```diff
@@ -863,6 +863,129 @@ func Test_buildForwardingRules(t *testing.T) {
 			},
 			nil,
 		},
+		{
+			"default forwarding rules",
```

Comment:
The test name [seems to be used already](https://github.com/digitalocean/digitalocean-cloud-controller-manager/blob/749a63e79dc08d4f1ed7f66d7fa69a7c748d1b04/cloud-controller-manager/do/loadbalancers_test.go#L612)? Copy-pasta?